packages/admin-ui/src/combobox/combobox.style.ts

```diff
@@ -0,0 +1,75 @@
+import { style } from '@vtex/admin-ui-core'
+
+const height = 48
+const width = '346px'
```

Comment:
Does this mean that the with wont be responsive?

Reply:
Maybe you can change it to `rem`?
```
height = '3rem'
width = '21.625rem'